angr/state_plugins/pitree/pitree.py

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/python
+
+"""
+pitree: cloneable paged interval tree
+
+Copyright 2017 Camil Demetrescu
```

Comment:
Do we have to do anything special for this? Should we append the license here to our LICENSE file? etc

Reply:
Need to move this out... Will contact the author sometime this week.